package.json

```diff
@@ -4,10 +4,15 @@
   "description": "This is a Workshop about Cypress",
   "main": "index.js",
   "scripts": {
-    "test": "cypress run",    
+    "delete:reports": "rmdir -f cypress\\reports",
```

Comment:
here you should use linux format as the ci is using an SO linux, locally you can use the git bash console to run the tests to avoid get failures in your windows machine, also the path should be in linux format